### DISCUSSION TITLE
Unable to find freetzmount in webif

### DISCUSSION THREAD

**Discussion by donacarr** [Freetz-NG/freetz-ng#367] [Allgemein] [answered]
After I update my 7590 from freetz to freetz-ng I cannot find anymore freezmount in webif.
I need to set a label for my USB disks so that the partitions can always be found under the same mount points.
Thank you for your help.

**Comment by fda77** [comment, marked as answer]
Its no longer supported since some FritzOS versions, use udevmount

**Comment by donacarr** [comment]
Thank you @fda77.
The discussion may be closed

**Reply by fda77** [reply]
Github can only this select-answer or lock-conversation
